src/components/tile-select/tile-select.component.js

```diff
@@ -132,9 +133,14 @@ const TileSelect = ({
                 )}
 
                 {titleAdornment && (
-                  <StyledAdornment>{titleAdornment}</StyledAdornment>
+                  <StyledAdornment
+                    hasAdditionalInformation={!!additionalInformation}
+                  >
+                    {titleAdornment}
+                  </StyledAdornment>
                 )}
               </StyledTitleContainer>
+              {additionalInformation && <div>{additionalInformation}</div>}
```

Comment:
Is the div necessary here?

Reply:
The content needs to be block level so that it achieves the layout in the design specs and a `div` ensures we won't have any invalid HTML markup. If we don't have a `div` we'd have to rely on the implementation teams to compose the content with a block level wrapper every time to achieve it